Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryBuilder.java

```diff
@@ -715,6 +743,10 @@ public Element resolve(String identifier) {
         return translatedLibrary.resolve(identifier);
     }
 
+    public List<Pair<String, Object>> resolveCaseIgnored(String identifier) {
```

Comment:
It's not clear what List<Pair<String, Object>> is getting us here? Shouldn't we be declaring a class rather than using the generic Pair<String, Object>?